app/src/main/res/values/array.xml

```diff
@@ -422,4 +422,11 @@
         <item>playlists</item>
     </string-array>
 
+    <string-array name="category_sponsor_skip_choices">
+        <item>Off</item>
+        <item>Manual</item>
+        <item>Automatic</item>
+    </string-array>
+
+
```

Comment:
The strings are not currently translatable. Please create one string array `sb_skip_options_values`, that contains `off`, `manual` and `automatic` as items (so the current one basically, but lowercased), and a second string array that's called `sb_skip_options` with `@string/all`, `@string/manually` and `@string/off`, and use them respectively in the preferences `entryValues` and `entries`.